2021/Miglior/requirements.txt

```diff
@@ -0,0 +1,29 @@
+Brotli==1.0.9
```

Comment:
Come vedi dalla lista di dipendenze, per fare un progetto tutto sommato limitato con queste dipendenze e' quasi impossibile assicurarsi che tutto funzioni per il meglio, non ci siano vulnerabilità' in nessuno dei componenti, che all'aggiornare anche di uno solo dei componenti non si si spacchi nulla. Insomma fosse stato un tool reale e non un progettino quests cosa avrebbe richiesto la riscrittura da capo del tool